### PR TITLE
Quiz support on Portal

### LINKED ISSUE #56: Adding Quiz as a destination

### PR DESCRIPTION
**Pull request by sudheshna-donthineni**
Added Quiz as a destination.  
- Added the Quiz URL for both staging and prod.
- Added the Quiz API key. 
- Also, multiple entry is not allowed if the destination is a Quiz. 

For testing:
- Have created a dummy session.
http://localhost:8080/?sessionId=test-quiz
https://staging-auth.avantifellows.org/?sessionId=test-quiz

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,17 +1,23 @@
 ### setting up AWS SQS
-VUE_APP_AWS_SQS_ACCESS_KEY=''
-VUE_APP_AWS_SQS_SECRET_KEY=''
-VUE_APP_AWS_REGION=''
-VUE_APP_AWS_SQS_URL=''
+VITE_APP_AWS_ACCESS_KEY_ID=''
+VITE_APP_AWS_SECRET_KEY=''
+VITE_APP_AWS_REGION=''
```
Comment (deepansh96) [inline .env.example:4]: why did you remove `sqs` from these names?

Comment (sudheshna-donthineni) [reply .env.example:4]: I actually hadn't updated the .env file in a while, it seemed. In the github workflow and the github env variables, this is how it is stored.
